apps/oracles/cex-price-feeds/src/fetch_prices.rs

```diff
@@ -56,24 +56,24 @@ impl SymbolsData {
     }
 }
 
-pub async fn get_prices(resources: &ResourceData) -> Result<PairsToResults> {
+pub async fn get_prices(resources: &ResourceData, timeout_secs: u64) -> Result<PairsToResults> {
     let symbols = SymbolsData::from_resources(&resources.symbols)?;
 
     let futures_set = FuturesUnordered::from_iter([
-        fetch::<BinancePriceFetcher>(&symbols.binance, None),
-        fetch::<BinanceUsPriceFetcher>(&symbols.binance_us, None),
-        fetch::<BitfinexPriceFetcher>(&symbols.bitfinex, None),
-        fetch::<BitgetPriceFetcher>(&[], None),
-        fetch::<BybitPriceFetcher>(&[], None),
-        fetch::<CoinbasePriceFetcher>(&symbols.coinbase, None),
-        fetch::<CryptoComPriceFetcher>(&[], None),
+        fetch::<BinancePriceFetcher>(&symbols.binance, None, timeout_secs),
+        fetch::<BinanceUsPriceFetcher>(&symbols.binance_us, None, timeout_secs),
+        fetch::<BitfinexPriceFetcher>(&symbols.bitfinex, None, timeout_secs),
+        fetch::<BitgetPriceFetcher>(&[], None, timeout_secs),
+        fetch::<BybitPriceFetcher>(&[], None, timeout_secs),
+        fetch::<CoinbasePriceFetcher>(&symbols.coinbase, None, timeout_secs),
+        fetch::<CryptoComPriceFetcher>(&[], None, timeout_secs),
         // fetch::<GateIoPriceFetcher>(&[], None),
```

Comment:
Should we not add the `timeout_secs` parameter here as well, just for completeness' sake?

```suggestion
        // fetch::<GateIoPriceFetcher>(&[], None, timeout_secs),
```